### PR TITLE
chore: test against node v6

### PR DESCRIPTION
**Pull request by nfischer**
Looks like node v6 just got released. Just tried this out via nvm and saw that one of our tests broke for node v6, so I fixed that test.

Not sure if appveyor can test against node v6 yet, but I added it to the mix anyway.

### CODE REVIEW

**Comment by nfischer** [comment]
Update: I removed node v6 from appveyor, since it doesn't seem to be supported yet.

**Comment by ariporad** [comment]
Oops... I didn't see this one when opening #434. I'll close that one.

**Comment by ariporad** [comment]
Oh, I could review this, couldn't I?

File: test/pwd.js [hunk 1]
```diff
@@ -23,6 +23,6 @@ var _pwd = shell.pwd();
 assert.equal(_pwd.code, 0);
 assert.ok(!_pwd.stderr);
 assert.equal(shell.error(), null);
-assert.equal(path.basename(_pwd), 'tmp');
+assert.equal(path.basename(_pwd.toString()), 'tmp');
```
Comment (ariporad) [inline test/pwd.js]: ? Why is this here? Seems unrelated.

**Comment by nfischer** [comment]
Yup, feel free to merge if this is good

File: test/pwd.js (hunk 1 above)
Comment (nfischer) [inline test/pwd.js]: It fails without this. In node v6, I believe `basename()` requires a regular string. If you can get it to pass without this, feel free to change as necessary

Comment (ariporad) [inline test/pwd.js]: Kk

**Comment by ariporad** [comment]
LGTM.

Being real smart today I am.
